src/firebase/auth.js

```diff
@@ -58,17 +58,57 @@ export const useFirebaseAuth = () => {
   }
 }
 
+// export const signInWithGoogle = async (authUser, updateUserData) => {
```

Comment:
Never leave those comments in our Base Stack it should be kept exceptionally clean

If leaving comments it should be to help developers work with our stack